pkg/configtx/application.go

```diff
@@ -234,9 +234,9 @@ func (c *ConfigTx) AnchorPeers(orgName string) ([]Address, error) {
 	return anchorPeers, nil
 }
 
-// AddApplicationOrg adds an organization to an existing Application configuration.
-// Will not error if organization already exists.
-func (c *ConfigTx) AddApplicationOrg(org Organization) error {
+// SetApplicationOrg adds an organization to an existing Application configuration.
+// If an Organization already exist in current configuration, its value will be overwritten.
```

Comment:
`If the application org already exists, its value will be overwritten.`

Reply:
done